### DISCUSSION TITLE
Removal of {---BREADCRUMB---} message in Pages and Updates

### DISCUSSION THREAD

**Discussion by rwjework** [e107inc/e107#5101] [Q&A Core]
Hi All, I was using e107 2.3.0 with the standard  Bootstrap 3 theme. When I updated to 2.3.2 I get {---BREADCRUMB---} at the top of all the pages. Do you know how I get rid of this? I see another response about "Quoting them out" Do I need to edit the standard theme code in 2.3.2 and what files do I do this in? Secondly after updating I am getting the check for updates spinning every time I enter the admin area. I have tried it with the preferences "Check for updates Once Per Day" on and off and it seems to be checking every time I enter the main admin landing page. Is there an easy way of switching this off? Thanks

**Comment by Jimmi08** [comment]
Normally. - {---BREADCRUMB---} is in layouts/ layout html files.  It should be parsed as empty or display actual breadcrumbs.  
But from the look at the core bootstrap3 theme - I don't see this there.  Report this as bug, it is very weird.

**Reply by rwjework** [reply]
Thanks Jimmi08.
I could not see where it could be changed.
<img width="379" alt="Issue with Breadcrumb" src="https://github.com/e107inc/e107/assets/11871882/fcb4e0a2-4de6-41f1-bc56-0d794f63c863">
This is what it looks like. Where do I report it as a bug? Thanks for your response. The other issue with the checking for updates all the time seems to be slowing the site down considerably. Would appreciate anyone's help on that one as well. Just wondering where to write out the update script. Thanks again
